site/content/en/docs/Examples/_index.md

```diff
@@ -22,6 +22,7 @@ These are all examples of simple game server implementations, that integrate the
 
 - {{< ghlink href="examples/simple-udp" >}}Simple UDP{{< /ghlink >}} (Go) - simple server and client that send UDP packets back and forth.
 - {{< ghlink href="examples/simple-tcp" >}}Simple TCP{{< /ghlink >}} (Go) - simple server that responds to new-line delimited messages sent over a TCP connection.
+- {{< ghlink href="examples/simple-game-server" >}}Simple gameserver{{< /ghlink >}} (Go) - simple server that responds to TCP connections or UDP packets on the same port.
```

Comment:
Please add release guards to this new link.

Reply:
How do I do that/Where can I read about how to do that?

Reply:
https://agones.dev/site/docs/contribute/#within-a-page :+1: